packages/graph/hash_graph/lib/graph/src/store/postgres/query/table.rs

```diff
@@ -131,9 +131,9 @@ pub enum Entities<'p> {
     RecordId,
     DecisionTime,
     TransactionTime,
-    // TODO: Remove when adjusting structural queries
-    //   see https://app.asana.com/0/0/1203491211535116/f
-    LowerTransactionTime,
+    // TODO: Remove when correctly resolving time intervals in subgraphs.
+    //   see https://app.asana.com/0/0/1203701389454316/f
+    ProjectedTime,
```

Comment:
Does the documentation for "projected time" exist anywhere else? I don't think the name is very intuitive, I get the choice of "projection" from a mathematical point of view but I feel like we could improve accessibility of the code by going for things like "pinned time axis" and such.

At the very least I'd want these terms explained somewhere (and I can't find that at the moment)

Reply:
There is a short documentation in `shared/identifier/time/axis.rs`:
```rust
/// Time axis for the projected image time.
///
/// This is used as the generic argument to time-related structs.
#[derive(Debug, Default, Copy, Clone, PartialEq, Eq, Hash)]
pub struct ProjectedTime;
```

Reply:
Gotcha, can we try and explain it a *lot* more. Explaining "projected time" by saying it's "the axis for the projected image time" isn't reallyyy an explanation

Reply:
I added a more in-depth documentation in 3ddd19f27775101a499c922b45080ece2489a687 describing the `TimeProjection` and it's axes. I linked the `ProjectedTime` axis documentation to that documentation.